src/problems/richardsmitchellettesudoku.clj

```diff
@@ -0,0 +1,58 @@
+(ns problems.richardsmitchellettesudoku)
+
+(defrecord State [board])
+
+(defn getSection [board box] (nth board (int (/ box 3))))
+(defn getBox [board box] (nth (nth board (int (/ box 3))) (rem box 3)))
```

Comment:
Whenever you get into `(nth (nth …))` things get hard to read. It's also weird that you have `(nth board (int (/ box 3)))` here, which is the definition of `getSection`, right? Why not call `getSection` in `getBox`?

You've then got *crazy* piles of stacked `nth`s in the next two functions, that *really* don't make any sense.